tests/helm/helm_template_test.go

```diff
@@ -0,0 +1,132 @@
+/**
```

Comment:
I still need to add the logic to run these tests.

Reply:
Meaning you have run them locally and they pass, but there is nothing to trigger them automatically yet?